src/myuplink/auth_abstract.py

```diff
@@ -21,10 +21,11 @@ async def request(self, method, url, **kwargs) -> ClientResponse:
             headers = {}
         else:
             headers = dict(headers)
+            kwargs.pop("headers")
```

Comment:
I'm not sure what this line is doing, can you help? :)

Reply:
If there is a headers keyword in incoming **kwargs, there will be two such keywords in the subsequent call to request below.